### DISCUSSION TITLE
Final project vision

### DISCUSSION THREAD

**Discussion by nasserkessas** [Pokemon-PythonRed/Pokemon-PythonRed#111] [Q&A] [answered]
Hello @TurnipGuy30,

I was wondering what you imagined the final project to look like. I noticed you had a commented line importing Pygame so I was wondering whether this will be shifted from the console in the future. There is a lot of functionality that simply cannot be implemented with the current platform like correct encounters, picking up items and engaging with other trainers.

If you are planning to shift the project over, when do you think you'll start this?

**Comment by riverpaper** [comment, marked as answer]
The current plan is to replicate *Pokémon Red* in a text-adventure style until the Mt. Moon exit, after which custom content is present, such as more trainers and areas. However, I understand this will likely take a long time, and I'm first making sure that game mechanics work as intended. I also don't have much time to work on personal projects such as this.

You've mentioned the `pygame` module. I wanted to use this for music, but I don't know how to best store music files in a GitHub repository. I also don't know how to use `pygame`'s `mixer`; I previously planned to use `winsound` but I've come across people using Linux whilst contributing.

In conclusion, there's a lot I'm unsure of. I started this project as a way to learn both Python and GitHub, and I don't want to stop it, but I'm not too sure how to continue.

**Reply by nasserkessas** [reply]
I actually had an idea of how to implement trainers into the game; they could be added as options in the menu and removed when battled.
Thanks for the clarifications though.

There's a bit of a problem with picking up items though, I can't think of a way to include them so they might have to be excluded.
